data/json/items/melee/unarmed_weapons.json

```
@@ -119,5 +119,59 @@
     "flags": [ "STAB", "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE" ],
     "price": 10000,
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 14 ] ]
+  },
+  {
+    "id": "katar",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "FIST_WEAPONS" ],
+    "symbol": "!",
+    "color": "dark_gray",
+    "name": { "str": "katar" },
+    "looks_like": "knuckle_katar",
+    "description": "A steel brace with a metal blade at the end, designed to allow the user to stab and slash with the same motions as a punch.",
+    "material": [ "steel" ],
+    "volume": "750 ml",
+    "cutting": 20,
+    "price_postapoc": 500,
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 12 ] ],
+    "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "spiked_boxing",
+    "category": "weapons",
+    "weapon_category": [ "FIST_WEAPONS" ],
+    "name": { "str": "spiked boxing gloves" },
+    "description": "Boxing gloves with nails driven through the inside.  Not exactly ring-legal, but there are no more referees.",
+    "weight": "464 g",
+    "volume": "1 L",
+    "price": 4500,
```

```suggestion
    "price": "45 USD",
     "price_postapoc": "450 cent",
```
String prices aren't really mandatory, just included it in the process of adding a suggestion for fixing lack of barter value.